portality/tasks/journal_in_out_doaj.py

```diff
@@ -83,16 +85,18 @@ def prepare(cls, username, **kwargs):
         journal_ids = kwargs.get("journal_ids")
 
         params = {}
-        params["set_in_doaj__journal_ids"] = journal_ids
-        params["set_in_doaj__in_doaj"] = kwargs.get("in_doaj")
+        cls.set_param(params, "journal_ids", journal_ids)
+        cls.set_param(params, "in_doaj", kwargs.get("in_doaj"))
 
-        if params["set_in_doaj__journal_ids"] is None or params["set_in_doaj__in_doaj"] is None:
+        if journal_ids is None or kwargs.get("in_doaj") is None:
```

Comment:
journal_ids could be `[]` as well, which would not be valid. Perhaps best to just ask `if journal_ids or kwargs.get("in_doaj") is None`.